webrtc-stats.html

```diff
@@ -2978,6 +2978,11 @@ <h2>
                   relevant candidate would be <code>"wifi"</code>, even when the next hop is over a
                   cellular connection.
                 </div>
+                <p class="fingerprint">
+                  This reveals information that would otherwise not be available to web pages,
```

Comment:
TBH I don't think this is a big issue for fingerprinting (one can already look up the service provider based on the IP and make a good guess at connection type), but it might be a more significant issue for people trying to use VPNs.